e2e-tests/README.md

```diff
@@ -0,0 +1,189 @@
+End-to-End Tests for Socorro
+=============================
+
+Continuous Integration
+----------------------
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.stage.saucelabs.svg?label=stage)](https://webqa-ci.mozilla.com/job/socorro.stage.saucelabs/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.saucelabs.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod.saucelabs/)
+
+
+This directory holds Socorro client-based end-to-end tests which is why they're different than the rest of the code in this repository.
+
+To review the specific Python packages the tests utilize, please review `e2e-tests/requirements.txt`.
+
+Set up and run Socorro tests
+-----------------------------
+
+Review the documentation for [pytest-selenium][pytest-selenium] and decide which browser
+environment you wish to target.
+
+We suggest using a different virtual environment for these tests than the
+rest of Socorro so you're not mixing requirements:
+
+	$ mkvirtualenv socorro-tests
+	$ pip install -r requirements.txt
+
+An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads].
+
+If you have multiple versions of Firefox installed, you can specifiy a specific one by using the `--firefox-path <path to firefox binary>` flag.
+
+___Running tests against localhost___
+
+	$ py.test --driver Firefox --base-url http://localhost:8000 tests/
+
+___Running the tests on stage___
+
+	$ py.test --driver Firefox tests/
+
+___Running tests against production___
+
+	$ py.test --driver Firefox --base-url https://crash-stats.mozilla.com tests/
+
+___Running specific tests___
+
+You can run tests in a given file::
+
+    $ py.test --driver Firefox tests/desktop/test_search.py
+
+You can run tests that match a specific name:
+
+    $ py.test --driver Firefox -k test_search_for_unrealistic_data
+
+You can run tests whose names match a specific pattern:
+
+    $ py.test --driver Firefox -k test_search
+
```

Comment:
I think it's worthwhile to include it. I'm going to keep trying to use it because it's so much much faster and it doesn't pop-up annoying windows whilst the tests are running. 

We ought to point out that this is NOT what's used in our cloud infrastructure (e.g. jenkins).